src/reducers/profile-view.js

```diff
@@ -305,9 +304,11 @@ function waitingForLibs(state: Set<RequestedLib> = new Set(), action: Action) {
 function selection(
   state: ProfileSelection = { hasSelection: false, isModifying: false },
   action: Action
-) {
+): ProfileSelection {
   // TODO: Rename to timeRangeSelection
   switch (action.type) {
+    case 'VIEW_PROFILE':
+      return { hasSelection: false, isModifying: false };
```

Comment:
Maybe a comment explaining this is needed for when we switch profiles ?

Reply:
Oof, this shouldn't be in here actually, as the invalidation is a follow-up. I don't think I mentioned that anywhere as you started reviewing before I was able to write up more info on this PR. I'm removing the line from this PR.